pitop/processing/algorithms/faces/face_detector_haarcascade.py

```diff
@@ -0,0 +1,121 @@
+import os
+import cv2
```

Comment:
use `import_opencv` from `pitop.processing.utils.vision_functions` since opencv might not be installed... check `pitop/processing/algorithms/line_detect.py` to see how it's used